.github/workflows/build_wheels.yml

```diff
@@ -37,6 +37,31 @@ jobs:
           CIBW_PRERELEASE_PYTHONS: True
           CIBW_TEST_EXTRAS: test
           CIBW_TEST_COMMAND: pytest {package}/tests
+      - uses: actions/upload-artifact@v3
+        with:
+          path: ./wheelhouse/*.whl
+
+  build_wheels_macos:
+    name: Wheel for MacOS-${{ matrix.cibw_python }}-${{ matrix.cibw_arch }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-11]
+        cibw_python: ["cp38-*", "cp39-*", "cp310-*", "cp311-*"]
```

Comment:
Why no 3.7 wheel for OS X, when we have one for Linux?

Reply:
They don't work for macOS:

https://cibuildwheel.readthedocs.io/en/stable/